### DISCUSSION TITLE
Add basic information about the service/container.

### DISCUSSION THREAD

**Discussion by hickey** [tteck/Proxmox#1114] [Ideas]
When the script creates a container instance on Proxmox, it already creates some notes for that instance. The following demonstrates what I am referring to:

![image](https://user-images.githubusercontent.com/39609/221923098-88da050c-cdb3-47d3-b343-42b6a1f01312.png)

It would be useful to also include notes here for the service that is being installed. Specifically how to connect to the service and the default login credentials if they are known. This would allow a person to install the service from your scripts and then look at the notes to connect up to it and start using it. Instead of having to search the net to learn enough about the service to get connected. 

This would also be a useful thing if there are any special considerations concerning the service that should be noted.

**Comment by hickey** [comment]
I have created a PR that explores updating the scripts to include more information on how to connect and log into the service. Here is a preliminary example: 

![image](https://user-images.githubusercontent.com/39609/222048703-87bc0a40-51b9-4224-9909-a3ae9a4417a3.png)

I am updating the generated code to move the buy me coffee chicklet back to the original location so that it is more visible. In this example it disappeared "below the fold" and many users may not see it. Ideas on how to organize the additional information is certainly welcome. 

I have also moved the generation of the text for the notes to a function so that it will be easy to identify/update and allow more programmatic generation of the text.

**Reply by hickey** [reply]
PR is #1115

**Reply by hickey** [reply]
I changed over to using `<code>` instead of `<pre>` so that everything would be inline. Here is the result.

![image](https://user-images.githubusercontent.com/39609/222050727-f6a80480-a821-491b-9528-d4a3a011d26f.png)

I actually like the look of leaving the buy me coffee chicklet at the bottom of the text. But I could be convinced to move it up so help support the project.

**Reply by hickey** [reply]
I generated the text with the chicket moved up. 

![image](https://user-images.githubusercontent.com/39609/222051271-f7644dff-e3d8-46ad-b85b-e4f794668796.png)

Now people can visualize and consensus can be reached for how it should look.

**Comment by tteck** [comment]
I appreciate it, but it's a feature I'm not interested in.

**Reply by hickey** [reply]
Just curious as to why not? I have used a number of your scripts to try out software that I normally would not bother with. Your scripts make it easy to get a container up and running of the software, but sometimes I have had to dig quite a bit to find where a service is running and there have been a couple it has almost been painful trying to track down the initial login credentials. For such a simple update it seems that it would benefit the community and lower the bar for users just starting out with Proxmox.

**Reply by tteck** [reply]
The feature under discussion has been debated several times in the forum, and each time it has been concluded that the instructions contain all the required initial login credentials, along with port information, making the process straightforward rather than difficult. Since initial login credentials are usually temporary, and users are encouraged to change their passwords after their first login for security reasons, adding temporary information may not be very useful or valuable. 

If you happen to notice any missing information in the instructions, my suggestion would be to create an issue to bring it to my attention, so that it can be addressed and fixed.
